shell/platform/darwin/macos/framework/Headers/FlutterViewController.h

```diff
@@ -20,19 +23,15 @@ typedef NS_ENUM(NSInteger, FlutterMouseTrackingMode) {
 };
 
 /**
- * Controls embedder plugins and communication with the underlying Flutter engine, managing a view
- * intended to handle key inputs and drawing protocols (see |view|).
- *
- * Can be launched headless (no managed view), at which point a Dart executable will be run on the
```

Comment:
Can this no longer be headless? I was curious why this got removed.

Reply:
This comment was cruft from an API fix that already happened. The original FDE APIs had grafted headless mode onto the view controller, which made no sense. That has since been fixed to align the overall API with iOS, where there's a FlutterEngine that you use directly for headless execution. I forgot to update this comment when I made that change, and noticed it while doing this rename.